### ISSUE TITLE
[Minor] Auto raid mode changes modes unnecessarily

### ISSUE THREAD

**Issue by askebv** [Heishe/idleraiders-second-run#93]
Say you're currently in raid mode, and all raiders are in that raid. Then if you change to auto-raid mode, it quickly changes to the farm map, and then back to the raid (which effectively resets the raid).

It probably shouldn't do anything if you're already in the appropriate raid.

It's not really an issue, per say - but I just wanted to mention it.

**Comment by askebv** [comment]
Hmm, at the time I posted this, I found it to quickly start the raid again. That was on Burning Cave.

But now I tried it again, this time on Frozen Coast, and it still stopped the raid (which I wouldn't expect) but instead of starting the raid shortly after, it also initialized the waiting/resting period that auto-raid has between raids:

![image](https://i.gyazo.com/64b26561adc180b312a22c3e7d12458c.gif)

EDIT: Just updated to Beta Patch 3 - and really like the "Resting" label that you added on the auto-raid mode!
